src/System.IO.Pipes/tests/System.IO.Pipes.Tests.csproj

```diff
@@ -37,12 +35,14 @@
     <Compile Include="NamedPipeTests\NamedPipeTest.CrossProcess.cs" />
     <Compile Include="NamedPipeTests\NamedPipeTestBase.cs" />
     <Compile Include="NamedPipeTests\NamedPipeTest.Read.cs" />
+    <Compile Include="NamedPipeTests\NamedPipeTest.RunAsClient.cs" Condition="'$(TargetGroup)' == ''"/>
```

Comment:
Nit: Whenever we have > 1 test file that falls into this category, we usually just group them in an ItemGroup that has this condition, as opposed to every Item that qualifies having the condition.